docs/source/faq.md

```diff
@@ -115,6 +115,33 @@ It allows customization of the various parameters. An environment variable
 has precedence over the value defined in `ucx.conf`.
 The file can be created using `ucx_info -Cf`.
 
+#### Build user application with UCX
+
+UCX supports metainformation subsystem based on `pkg-config` tool. To add UCX support
+into application, output from `pkg-config` tool may be used in Makefile:
+```
+program: program.c
+        cc program.c $(pkg-config --cflags --libs ucx)
+```
+To build static application with UCX supporting just specific network transport
+modules (e.g. CMA and KNEM), the following command can be used:
+```
+program: program.c
+        cc -static program.c $(pkg-config --cflags --libs --static ucx-cma ucx-knem ucx)
+```
+Currently supported modules are: `ucx-cma`, `ucx-ib`, `ucx-knem`, `ucx-xpmem` and `ucx-rdmacm`.
+Transports `TCP`, `SHM` and `self` are built-in into UCT and don't need additional
+compilation actions.
+
+> **IMPORTANT NOTE:** static module ucx-ib depends on static libs `libnl` and
```

Comment:
may add this note to the above table

Reply:
hmmm, first we notice ucx-ib module and after this add important note. I think this location is more "gracefully"

Reply:
ok, BUT seems the table is not rendered as part of the note:
![image](https://user-images.githubusercontent.com/2255631/163983966-6592e2c9-86f0-4bc1-ba2c-6e6d36b47e37.png)

Reply:
yes, but it seems this is markdown limitation

Reply:
may be remove quoting from note?

Reply:
yes, make it a sub-header

Reply:
> yes, make it a sub-header

done

Reply:
what about pkg for AMD ROCM, Nvidia CUDA, ugni ? I would expect it should not be very hard to add pkg conf for this transports ?

Reply:
adding pkg-configs for these transport itself is not an issue, but static dependency libs could produce a lot of investigations. we will look at this for next releases

Reply:
I would suggest to include those now and just indicate that static build is not supported for those libraries.

Reply:
pkg-config files for transports useful for static build only. in dynamic mode UCX loads it in runtime, no need to make any special tricks for runtime

Reply:
But the tool is can be used for dynamic as well ? I think what your are saying that today it is only used for static builds.

Reply:
dynamic link suppose to use pkg-config `ucx` or `ucx-uct`, no additional modules should be added